src/utils.ts

```diff
@@ -84,6 +90,40 @@ export function functionalStyle<T>(
   return style;
 }
 
+/**
+ * Checks the style added to a component and if is a id function , gets the resulting value
+ */
+export function functionalStyleId(
+  elementStyle: CSSFuncPropsId,
+  id: string,
+): CSSProperties {
+  let style: CSSProperties = {};
+  for (const key in elementStyle) {
+    if (typeof elementStyle[key] === 'function') {
+      style[key] = elementStyle[key](id);
```

Comment:
please call it with an object `elementStyle[key]({id})` to be ready in case we add other parameters in the future

Reply:
fixed and i used an old function "functionalStyle" used to style scatterSeries points (now we can specifiy T )